CHANGELOG.md

```diff
@@ -91,6 +91,7 @@
 - Refactored `quoted_identifier_to_snowflake_type` to avoid making metadata queries if the types have been cached locally.
 - Improved `pd.to_datetime` to handle all local input cases. 
 - Create a lazy index from another lazy index without pulling data to client.
+- Display a clearer error message when `Index.names` is set to a non-like-like object.
```

Comment:
```suggestion
- Displayed a clearer error message when `Index.names` is set to a non-like-like object.
```

Reply:
I think display is the correct tense here - `displayed` implies it's something that happened only once in the past, not something like an error that can happen again

Reply:
we use past tense everywhere in this file and the only exceptions are this one and the previous one.

Reply:
Yeah, English is weird. I think everything else is past tense because the action is a one-time fix forever, but the error message here is displayed every time the user runs into the error - so it should be present tense. Same applies for the line above - lazy index can be created multiple times.